src/sentry/utils/raven.py

```diff
@@ -34,16 +35,26 @@ def is_enabled(self):
             return False
         return settings.SENTRY_PROJECT is not None
 
-    def capture(self, *args, **kwargs):
```

Comment:
This was explicitly moved out of `capture()` and into `send()` because we should send _all_ errors to an upstream since there's no risk of recursion and doesn't need to be filtered out.